platform/android/java/lib/build.gradle

```diff
@@ -79,15 +79,6 @@ android {
         editorDev.jniLibs.srcDirs = ['libs/tools/dev']
```

Comment:
You need an additional entry for `editorRelease`. This is likely the source of the crash, as the generated apk doesn't have the proper shared libraries since it doesn't know where to pick it from.